tests/lib/mcp/mcp-server.js

```diff
@@ -55,6 +56,14 @@ describe("MCP Server", () => {
 		// Note: must connect server first or else client hangs
 		await mcpServer.connect(serverTransport);
 		await client.connect(clientTransport);
+
+		sinon
+			.stub(process, "emitWarning")
+			.withArgs(sinon.match.any, "ESLintIgnoreWarning");
```

Comment:
```suggestion
			.withArgs(sinon.match.any, "ESLintIgnoreWarning")
			.returns();
		process.emitWarning.callThrough();
```


To allow other potentially useful warnings to appear in the output.

Reply:
You are correct, if `callThrough()` or `returns()` is omitted, other unintended warnings are suppressed as well. I was not aware of this aspect of sinon.